spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/KinesisBinderEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,37 @@
+package org.springframework.cloud.stream.binder.kinesis;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+
+/**
+ * An {@link EnvironmentPostProcessor} that sets some common configuration properties (log config etc.,) for Kinesis
+ *  binder.
+ *
+ * @author Peter Oates
+ */
+public class KinesisBinderEnvironmentPostProcessor implements EnvironmentPostProcessor {
+
+	public final static String SPRING_KINESIS = "spring.kinesis";
+
+	public final static String SPRING_KINESIS_PRODUCER = SPRING_KINESIS + ".producer";
+
+	public final static String SPRING_KINESIS_CONSUMER = SPRING_KINESIS + ".consumer";
+
+	private static final String KINESIS_BINDER_DEFAULT_PROPERTIES = "kinesisBinderDefaultProperties";
+
+	@Override
+	public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
+		if (!environment.getPropertySources().contains(KINESIS_BINDER_DEFAULT_PROPERTIES)) {
+			Map<String, Object> kinesisBinderDefaultProperties = new HashMap<>();
+			kinesisBinderDefaultProperties.put("logging.pattern.console", "%d{ISO8601} %5p %t %c{2}:%L - %m%n");
+			kinesisBinderDefaultProperties.put("logging.level.kinesis.server.KinesisConfig", "ERROR");
+			kinesisBinderDefaultProperties.put("logging.level.kinesis.admin.AdminClient.AdminConfig", "ERROR");
+			environment.getPropertySources().addLast(new MapPropertySource(KINESIS_BINDER_DEFAULT_PROPERTIES, kinesisBinderDefaultProperties));
```

Comment:
This fully looks like copy-paste beast from Kafka Binder. 
Since it doesn't bring to Kinesis any gains, I'd prefer to remove it altogether.
At least until the real requirement.

Anyway all these `Environment` manipulations look hacky and can easily overrides user preferences unconditionally.